src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -95,5 +108,71 @@ internal abstract partial class AbstractFindUsagesService : IFindUsagesService
 
             return progressAdapter;
         }
+
+        private async Task<bool> TryFindLiteralReferencesAsync(
+            Document document, int position, IFindUsagesContext context)
+        {
+            var cancellationToken = context.CancellationToken;
+            cancellationToken.ThrowIfCancellationRequested();
+
+            var syntaxTree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
+            var token = await syntaxTree.GetTouchingTokenAsync(
+                position,
+                t => syntaxFacts.IsNumericLiteral(t) ||
+                     syntaxFacts.IsCharacterLiteral(t) ||
+                     syntaxFacts.IsStringLiteral(t),
+                cancellationToken).ConfigureAwait(false);
+
+            if (token.RawKind == 0)
+            {
+                return false;
+            }
+
+            // Searching for decimals not supported currently.
+            var tokenValue = token.Value;
+            if (tokenValue == null || tokenValue is decimal)
+            {
+                return false;
+            }
+
+            var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            var symbol = semanticModel.GetSymbolInfo(token.Parent).Symbol ?? semanticModel.GetDeclaredSymbol(token.Parent);
+            if (symbol is ILabelSymbol)
```

Comment:
Because Labels are already supported by the existing FAR system.  In VB you can have numeric labels.   In that case, FAR should really only find the real VB symbol references, and not matching literals.